.github/workflows/build-test-package-python.yml

```diff
@@ -0,0 +1,67 @@
+name: test, package
+
+on:
+  push:
+    branches: ['main']
+  pull_request:
+    branches: ['main']
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      # max-parallel: 6
+      matrix:
+        os: [ubuntu-latest]
+        python-version: ['3.7', '3.8', '3.9', '3.10']
```

Comment:
Can we add 3.11?

Reply:
Done in [2f2e814](https://github.com/InsightSoftwareConsortium/itk_cucim/commit/2f2e814bd3914bfebbeddcba2eb64d9ab16c461d).